src/element/Nip05.css

```diff
@@ -7,13 +7,63 @@
 
 .nip05 .nick {
     color: var(--gray-light);
+    font-weight: bold;
+    margin-right: .2em;
 }
 
 .nip05 .domain {
     color: var(--gray-superlight);
+    font-weight: bold;
+}
+
+.nip05 .domain[data-domain="snort.social"] {
+    background: var(--snort-gradient);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    background-clip: text;
+    text-fill-color: transparent;
```

Comment:
Can you add the repeated styles to a common class?

Reply:
[f1cd244](https://github.com/v0l/snort/pull/52/commits/f1cd244cb7aaaef72a512ea066a18491638d5f72)